### PR TITLE
Update libvorbis

### PR DESCRIPTION
**Pull request by Luni-4**
- Update `libvorbis`
- Update `vorbis-sys` version

Thanks in advance for your review! :)

### CODE REVIEW

**Comment by Luni-4** [comment]
@tomaka 

Gently ping :)

File: Cargo.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name = "vorbis-sys"
 description = "FFI for the libvorbis library"
 license = "MIT"
-version = "0.0.8"
+version = "0.0.9"
```
Comment (tomaka) [inline Cargo.toml]:
```suggestion
version = "0.1.0"
```

**Comment by Luni-4** [comment]
> Sorry! Could just bump to 0.1.0, then I'll release.

Don't worry, np! Thanks for your review! :)

**Comment by Luni-4** [comment]
Thank you! :)

**Pull request merged**